assets/css/style.scss

```diff
@@ -1,155 +1,149 @@
-			padding: 0 $gap 0 0;
-			margin: 0 0 $gap-large 0;
-			float: none;
-			width: auto;
-			position: relative;
-			text-align: center;
-
-			.wc-block-grid__product-link {
-				text-decoration: none;
-				border: 0;
-			}
+.wc-block-grid__product-link {
+	text-decoration: none;
+	border: 0;
+}
 
-			.wc-block-grid__product-image {
-				text-decoration: none;
-				margin-bottom: $gap;
-				display: block;
+.wc-block-grid__product-image {
+	text-decoration: none;
+	margin-bottom: $gap;
+	display: block;
 
-				img {
-					vertical-align: middle;
-					margin-left: auto;
-					margin-right: auto;
-				}
-			}
+	img {
+		vertical-align: middle;
+		margin-left: auto;
+		margin-right: auto;
+	}
+}
 
-			.wc-block-grid__product-title {
-				line-height: 1.2;
-				font-weight: 700;
-			}
+.wc-block-grid__product-title {
+	line-height: 1.2;
+	font-weight: 700;
+}
 
-			.wc-block-grid__product-title,
-			.wc-block-grid__product-price,
-			.wc-block-grid__product-rating {
-				margin-bottom: $gap-small;
-				display: block;
-			}
+.wc-block-grid__product-title,
+.wc-block-grid__product-price,
+.wc-block-grid__product-rating {
+	margin-bottom: $gap-small;
+	display: block;
+}
 
-			.wp-block-button {
-				margin: 0;
+.wc-block-grid__product-add-to-cart {
+	margin: 0;
+	word-break: break-word;
+	white-space: normal;
+
+	a {
+		word-break: break-word;
+		white-space: normal;
+		margin: 0;
+
+		&.loading {
+			opacity: 0.25;
+			padding-right: 2.618em;
+
+			&::after {
+				font-family: WooCommerce; /* stylelint-disable-line */
+				content: "\e01c";
+				vertical-align: top;
+				font-weight: 400;
+				position: absolute;
+				top: 0.618em;
+				right: 1em;
+				animation: spin 2s linear infinite;
 			}
+		}
 
-			.wc-block-grid__product-add-to-cart {
-				margin: 0;
-				word-break: break-word;
-				white-space: normal;
-
-				a {
-					word-break: break-word;
-					white-space: normal;
-					margin: 0;
-
-					&.loading {
-						opacity: 0.25;
-						padding-right: 2.618em;
-
-						&::after {
-							font-family: WooCommerce; /* stylelint-disable-line */
-							content: "\e01c";
-							vertical-align: top;
-							font-weight: 400;
-							position: absolute;
-							top: 0.618em;
-							right: 1em;
-							animation: spin 2s linear infinite;
-						}
-					}
-
-					&.added::after {
-						font-family: WooCommerce; /* stylelint-disable-line */
-						content: "\e017";
-						margin-left: 0.53em;
-						vertical-align: bottom;
-					}
-				}
-
-				.added_to_cart {
-					text-align: center;
-				}
-			}
+		&.added::after {
+			font-family: WooCommerce; /* stylelint-disable-line */
+			content: "\e017";
+			margin-left: 0.53em;
+			vertical-align: bottom;
+		}
+	}
 
-			.wc-block-grid__product-onsale {
-				font-weight: 700;
-				position: absolute;
-				text-align: center;
-				top: 0;
-				left: 0;
-				margin: 0;
-				background-color: #000;
-				color: #fff;
-				display: inline-block;
-				font-size: 0.75em;
-				line-height: 1;
```

Comment:
Removing this line makes the _Sale!_ label higher in Twenty Nineteen.

Before:
![image](https://user-images.githubusercontent.com/3616980/62635979-020a1380-b939-11e9-97ef-adc031d012a6.png)

After:
![image](https://user-images.githubusercontent.com/3616980/62636005-0cc4a880-b939-11e9-8828-1ff79d4f704f.png)